server/modules/auth/auth.controller.ts

```
@@ -0,0 +1,372 @@
+//     });
+
+//     res.status(201).json({
+//       success: true,
+//       data: { user: userWithoutPassword, token },
+//     });
+//   }
+// );
+
+export const register = asyncHandler(
+  async (
+    req: Request<{}, {}, RegisterBody>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)